pkg/kubemark/hollow_kubelet.go

```diff
@@ -208,5 +212,14 @@ func GetHollowKubeletConfig(opt *HollowKubletOptions) (*options.KubeletFlags, *k
 	c.SystemCgroups = ""
 	c.ProtectKernelDefaults = false
 
-	return f, c
+	// Instance config
+	i, err := options.NewKubeletConfigurationInstance()
+	if err != nil {
+		panic(err)
+	}
+
+	i.HostnameOverride = opt.NodeName
```

Comment:
I think we should set `i.Address` the same as `c.Address` as well (`0.0.0.0`, above in this file).